### PR TITLE
feat: tutorial on debugging Kotlin/JS

### PR DESCRIPTION
**Pull request by p7nov**

### CODE REVIEW

**Comment by p7nov** [comment]
Note: file name is changed. Don't forget to set up a redirect.

**Review by ilgonmic — APPROVED** [review]

File: docs/topics/js/js-debugging.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Most modern browsers provide tools that allow inspecting the page content and debugging the code that executes
+on it. Refer to your browser's documentation for more details.
+
+To debug Kotlin/JS in the browser:
+
+1. Run the project by calling one of the available `*Run` Gradle tasks, for example, `browserDevelopmentRun` or
+   `jsBrowserDevelopmentRun` in a multiplatform project.
+   Learn more in [this tutorial](running-kotlin-js.md#run-the-browser-target).
+2. Navigate to the page in the browser and launch its developer tools, for example, by right-clicking and
+   selecting the **Inspect** action. Learn how to [find the developer tools](https://balsamiq.com/support/faqs/browserconsole/)
```
Comment (koshachy) [inline docs/topics/js/js-debugging.md:21]: Not quite sure that this link is the best place we could lead to, but it's ok)

File: docs/topics/js/js-debugging.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+4. Click the file reference on the right to navigate to the corresponding line of code.
+   Alternatively, you can manually switch to the **Sources** tab, and find the file you need in the file tree. Navigating
+   to the Kotlin file shows you the regular Kotlin code (as opposed to minified JavaScript):
+
+   ![Debugging in Chrome DevTools](devtools-sources.png)
+   {width="600"}
+
+You can now start debugging the program. For example, set a breakpoint by clicking on one of the line numbers.
+The developer tools even support setting breakpoints within a statement. As with regular JavaScript code, any set
+breakpoints will persist across page reloads. This also makes it possible to debug Kotlin's main method which is executed
```
Comment (koshachy) [inline docs/topics/js/js-debugging.md:38]: If we're talking about the specific method, we can write `main()`.

File: docs/topics/js/js-debugging.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+   {width="600"}
+
+4. Click the file reference on the right to navigate to the corresponding line of code.
+
+You can now start debugging the program using the whole set of tools that the IDE offers: breakpoints, stepping, expression
+evaluation, [coroutine debugger](debug-coroutines-with-idea.md) and more. Learn more about [debugging in IntelliJ IDEA](https://www.jetbrains.com/help/idea/debugging-code.html).
+
+## Debug in Node.js
+
+If your projects targets Node.js, you can debug it right in this runtime.
```
Comment (koshachy) [inline docs/topics/js/js-debugging.md:80]:
```suggestion
If your projects target Node.js, you can debug them right in this runtime.
```

Comment (koshachy) [reply docs/topics/js/js-debugging.md:80]: Maybe it's better to rephrase in the singular.

File: docs/topics/js/js-debugging.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+To debug a Kotlin/JS application targeting Node.js:
+
+1. Build the project by running the `build` Gradle task.
+2. Find the resulting `.js` file for executing in Node.js in the `build/js/packages/your-module/kotlin/` directory inside
```
Comment (koshachy) [inline docs/topics/js/js-debugging.md:85]: Maybe it's better to leave only one gerund to make the sentence more readable.

**Review by koshachy — APPROVED** [review]
Everything is super clean and neat. Nice work!

File: docs/topics/js/js-debugging.md (hunk 1 above)
Comment (p7nov) [reply docs/topics/js/js-debugging.md:21]: Seb offered this as a single source of info about multiple browsers

File: docs/topics/js/js-debugging.md (hunk 3 above)
Comment (p7nov) [reply docs/topics/js/js-debugging.md:80]: Yep, I'd use singular here.

**Pull request merged**